draft-opsarea-rfc5706bis.md

```diff
@@ -563,6 +568,30 @@ This document does not describe interoperability requirements. As such, it does
+
+   Indicators of Compromise (IoCs) {{?RFC9424}} are relied upon by security operators to identify and block malicious activity. A New Protocol or Protocol Extension may have implications for the types, locations or availability of IoCs and Protocol Designers should outline such impacts. Similarly, to be most effective, IoCs are required to be shared at scale so consideration should be given to ensure this is possible. 
+
+   Security operators are required to understand, detect and counter malicious activity, and adapt to changing threats. New Protocol or Protocol Extension may change attacker activity, which would change how a security operator would detect and defend against such activity. As such, the protocol specification should consider any impact to the ability of attackers to perform malicious activities, such as Command and Control (C2) communications, network traversal or facilitation of exfiltration of data from the network. A New Protocol may offer different opportunities for performing such malicious activity or could prevent current techniques, which would impact how a security operator manages their network. 
+
+   As in {{sec-impact}}, Protocol Designers should also consider the impact of traffic levels and traffic patterns from a security perspective, for example, factoring in the ability to defend against malicious distributed denial-of-service (DDoS) attacks. Protocol Designers should also consider whether New Protocol could be used to create DDoS attacks, for example amplication attacks in DNS or NTP. Operators may require new tooling or methods for managing network traffic in response to a change in potential risk to ensure consistent availability and performance of the network. 
+
+   Many security operation tools are automated to detect threats, and implications of New Protocols and Protocol Extensions for such tooling should be captured, for example aiming to minimise the likelihood of false positives which can lead to operators to be desensitised to alerts when deploying a New Protocol.
+
+   In order to reduce the effects of compromise, security operators may also require digital forensics from the network or endpoints to enable effective incident response or threat hunting. Key sources of this information may be found in logs which could provide details of authentication or authorization events, network traffic or endpoint-detection events, for example. In particular, where a New Protocol changes the properties or topology of the network, Protocol Designers should consider the requirement for digital forensics and whether logs for the network, endpoints or any new intermediaries introduced are impacted positively or negatively. {{?I-D.ietf-quic-qlog-main-schema}} is an example of structured logging for network protocols, and new ways of providing logging should be considered when designing a New Protocol.  
```

Comment:
I don't disagree with this, but it's kind of abstract in what it's recommending.  What specific "new ways" of providing logs should designers consider?